lib/services/access_request.go

```diff
@@ -1257,6 +1264,49 @@ func (m *RequestValidator) Validate(ctx context.Context, req types.AccessRequest
 				return trace.Wrap(err)
 			}
 		}
+
+		// Validate kube request kinds.
+		// If request mode is defined, then any request for kube_cluster will be rejected.
+		isResourceRequest := len(req.GetRequestedResourceIDs()) > 0
```

Comment:
what happens if multiple roles back this access request?
Should we return an error only if none of them support the kind. If role doesn't support the kind Y, we won't use for access to resources of Kind Y